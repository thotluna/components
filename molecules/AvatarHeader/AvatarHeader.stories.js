import { AvatarHeader, options, styles } from './index'
import {
  getListTemplate,
  getTemplate,
  getOptionsArgTypes,
} from '../../helpers/storybook'

const Template = getTemplate(AvatarHeader, styles)
const ListTemplate = getListTemplate(AvatarHeader, styles)

export default {
  title: 'Molecules/AvatarHeader',
  component: AvatarHeader,
  args: {
    src: 'https://picsum.photos/id/1027/200/200',
    size: 'md',
    title: 'ThotLuna',
    subtitle: 'thotluna@gmail.com',
  },
  argTypes: {
    src: { control: 'text' },
    size: getOptionsArgTypes(options.size),
  },
}

export const Default = Template.bind({})

export const Size = ListTemplate.bind({})
Size.args = { items: options.size.map((size) => ({ size })) }
