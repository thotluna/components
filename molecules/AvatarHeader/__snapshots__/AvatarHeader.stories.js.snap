// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Molecules/AvatarHeader Default 1`] = `
<div
  style={
    Object {
      "alignContent": "flex-start",
      "display": "flex",
      "flexDirection": "column",
      "flexWrap": "wrap",
      "gap": "10px 30px",
      "height": "100%",
      "justifyContent": "flex-start",
      "maxHeight": "auto",
    }
  }
>
  <div
    className="container"
  >
    <img
      alt="Avatar"
      className="container size-md"
      src="https://picsum.photos/id/1027/200/200"
    />
    <div
      style={
        Object {
          "alignItems": "flex-start",
          "display": "flex",
          "flexDirection": "column",
          "justifyContent": "space-between",
        }
      }
    >
      <header
        className="headings color-base size-md weight-bold"
      >
        ThotLuna
      </header>
      <div
        className="container color-base size-md weight-bold"
      >
        thotluna@gmail.com
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Molecules/AvatarHeader Size 1`] = `
<div
  style={
    Object {
      "alignContent": "flex-start",
      "display": "flex",
      "flexDirection": "column",
      "flexWrap": "wrap",
      "gap": "10px 30px",
      "height": "100%",
      "justifyContent": "flex-start",
      "maxHeight": "auto",
    }
  }
>
  <div
    className="container"
  >
    <img
      alt="Avatar"
      className="container size-sm"
      src="https://picsum.photos/id/1027/200/200"
    />
    <div
      style={
        Object {
          "alignItems": "flex-start",
          "display": "flex",
          "flexDirection": "column",
          "justifyContent": "space-between",
        }
      }
    >
      <header
        className="headings color-base size-sm weight-bold"
      >
        ThotLuna
      </header>
      <div
        className="container color-base size-sm weight-bold"
      >
        thotluna@gmail.com
      </div>
    </div>
  </div>
  <div
    className="container"
  >
    <img
      alt="Avatar"
      className="container size-md"
      src="https://picsum.photos/id/1027/200/200"
    />
    <div
      style={
        Object {
          "alignItems": "flex-start",
          "display": "flex",
          "flexDirection": "column",
          "justifyContent": "space-between",
        }
      }
    >
      <header
        className="headings color-base size-md weight-bold"
      >
        ThotLuna
      </header>
      <div
        className="container color-base size-md weight-bold"
      >
        thotluna@gmail.com
      </div>
    </div>
  </div>
</div>
`;
